### DISCUSSION TITLE
Changing the vivado-on-silicon-mac-main folder location

### DISCUSSION THREAD

**Discussion by atahanhuyuk** [ichi4096/vivado-on-silicon-mac#35] [Q&A] [answered]
Hello,

I successfully completed the install. However I did it while the vivado-on-silicon-mac-main folder was in downloads. I want to change its location to documents but when I do so the launch vivado app does not work properly. If anyone knows how I can change the location, I would appreciate the help!

**Comment by atahanhuyuk** [comment, marked as answer]
Well just after posting this I found the solution. For anyone searching for this information:

Show the package contents of Launch Vivado
Right click on the executable and open with the IDE of your choice
Simply change the file destination of vivado-on-silicon-mac-main inside this file to whereever you want to relocate the file to
Save and enjoy!

**Comment by ichi4096** [comment]
There is no app anymore in the newest version of my scripts since it caused issues with file permissions. The installation path should not matter anymore now.
